realm/realm-library/src/androidTest/java/io/realm/RealmLinkTests.java

```diff
@@ -76,12 +95,16 @@ protected void setUp() {
         testRealm.commitTransaction();
     }
 
-    @Override
-    protected void tearDown() throws Exception {
-        testRealm.close();
+    @After
+    public void tearDown() {
+        if (testRealm != null) {
+            testRealm.close();
+        }
+        Realm.deleteRealm(realmConfig);
```

Comment:
This should happen automatically when we use the `TestRealmConfigurationFactory`